tests/general/KeywordBearTest.py

```diff
@@ -214,3 +214,22 @@ def test_keyword_regex(self):
             self.assertEqual(result[0].message, 'The line contains the keyword'
                                                 " 'Issue #123' which resulted "
                                                 'in a match with given regex.')
+
+    def test_wrong_language(self):
+        self.section.append(Setting('language', 'anything'))
+        annotation_bear_result_type = namedtuple('result', 'contents')
+        dep_results = {
+            'AnnotationBear': [
+                annotation_bear_result_type(
+                  'coalang specification for anything not found.'
+                  )
+            ]
+        }
+
+        text = ['# todo 123']
+
+        with execute_bear(self.uut, 'F', text,
+                          dependency_results=dep_results) as result:
+            self.assertEqual(result[0].diffs, {})
+            self.assertEqual(result[0].affected_code[0].start.line, 1)
+            self.assertEqual(len(result), 1)
```

Comment:
And there aren't any checks to make sure the message is logged? I mean, that's what the whole PR is about.